api/src/main/java/io/opencensus/common/Constants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016-17, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.common;
+
+/**
+ * Class holder for all common constants for the OpenCensus Java library.
+ */
+public final class Constants {
```

Comment:
What other constants will go in this class?  Is Constants.VERSION meant to be used by all users?

Reply:
Moved to internal and add @Internal annotation.

Reply:
I thought that we previously only used `Internal` for elements that were internal to the API library, to indicate that they weren't stable.  Since this field needs to be backwards compatible for the exporter, I would rather make it part of the API, but maybe change the class name.  I've seen other libraries expose a version constant.